### DISCUSSION TITLE
ESP32

### DISCUSSION THREAD

**Discussion by shamirtahir50** [Bodmer/TFT_eSPI#2274] [General]
Hey @Bodmer,

Which lcd driver I should use with ESP32 or whats the best controller for TFTs?

I want to use SPI because I am currently using 16-bit conections and I want to get rid of them.

I want it to be power effiecient as well.

**Comment by Bodmer** [comment]
That depends on your screen size and resolution requirements.

ST7789, SSD1963, ST7796 are supported. IPS screens are good for increased viewing angle range. ILI9488 screen are slower due to the need to send 3 bytes per pixel but are commonly available and cheap.

To reduce power you will need to be able to turn off the backlight LED.
